ambuda/views/proofing.py

```diff
@@ -514,10 +514,9 @@ def revision(project_slug, page_slug, revision_id):
     if not cur_revision:
         abort(404)
 
-    if prev_revision:
-        diff = _revision_diff(prev_revision.content, cur_revision.content)
-    else:
-        diff = _revision_diff("", cur_revision.content)
+    diff = _revision_diff(
+        prev_revision.content if prev_revision else "", cur_revision.content
```

Comment:
nit: I prefer the old version, see "Expression Complexity" here https://grugbrain.dev/

Reply:
I agree that we want to maintain simplicity, especially for a project where new contributors will be joining often. I thought this change wouldn't affect readability too detrimentally but if you feel it does then others must as well so I'll restore it.